src/js/components/accordion.js

```diff
@@ -70,7 +70,7 @@ Accordion.prototype.show = function ($button) {
  */
 Accordion.prototype.hideAll = function () {
   var self = this;
-  this.$('button').each(function () {
+  this.$('ul > li > button, .usa-accordion-button').each(function () {
```

Comment:
Ah! I see that you left this in for backwards compatibility. Totally okay to do this, but would like to know your thoughts on when we should remove this use case as we move towards `1.0`.